packages/model-viewer/src/features/animation.ts

```diff
@@ -101,6 +101,16 @@ export const AnimationMixin = <T extends Constructor<ModelViewerElementBase>>(
       this[$needsRender]();
     }
 
+    get currentTimeScale(): number {
```

Comment:
Could you add this to the docs and perhaps add/modify an animation example to demonstrate this?

Reply:
added docs & sample